material.css

```diff
@@ -12,6 +12,17 @@
     background: #F5F5F5;
 }
 
+.App-header {
+    padding: 8px;
+    height: 52px;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 1000;
+    box-shadow: 0px 2px 2px rgba(0,0,0,0.25);
```

Comment:
Can you decrease the shadow a bit? It looks a lot after scrolling down as it increases that time.

Reply:
Let me see what I can do.

Reply:
Fixed!